stf_functional_tests.yml

```diff
@@ -52,6 +52,21 @@
       import_role:
         name: client_side_tests
         tasks_from: test_e2e
+      
+    - name: "Test Alerts"
+      import_role:
+        name: test_alerts
+        tasks_from: first_test
+    
+    - name: "Test metrics retention"
+      import_role:
+        name: test_metrics_retention
+        tasks_from: metrics_retention_test
```

Comment:
Since there's just one task in the test_metrics_retention role, you can rename the task to main and remove the ``tasks_from`` line here.